main/main.cpp

```diff
@@ -3351,6 +3354,8 @@ int Main::start() {
 			if (parsed_pair) {
 				E = E->next();
 			}
+		} else if (E->get().contains("export")) {
+			ERR_FAIL_V_MSG(EXIT_FAILURE, "Please provide a valid preset when exporting, aborting.");
```

Comment:
What does this handle? Export options set preset with `E->next()->get()`.

Reply:
> Export options set preset with `E->next()->get()`.

I remembered it was for when user only use `exported` but not give it any more args.

Reply:
Ah yes, there is this `if (E->next())`, which makes `export-` only parsed when it's not the last argument. Still, your check is too broad. You could at least change it to `begins_with("--export-")`, so it matches only valid export commands.

Reply:
Good idea, Done